src/twisted/mail/test/test_mail.py

```diff
@@ -222,7 +223,7 @@ class FileMessageTests(TestCase):
     def setUp(self):
         self.name = "fileMessage.testFile"
```

Comment:
Assuming that the specifics of the name don't matter, [`SynchronousTestCase.mktemp()`](https://twistedmatrix.com/documents/current/api/twisted.trial.unittest.SynchronousTestCase.html#mktemp) will get you a path to use inside `_trial_temp`.

```suggestion
        self.name = self.mktemp()
```